archeryutils/handicaps/handicap_equations.py

```diff
@@ -355,15 +356,126 @@ def sigma_r(
     return sig_r
 
 
+def s_bar(
```

Comment:
As I put in my original comment, I understand separating for now for testing/comparison, but assuming negligible impact on the outputs I would look to replace the internals of `arrow_score` with that here.

Reply:
Changed mind, see review below post-handicap-refactor.